pages/status.tsx

```diff
@@ -93,14 +99,16 @@ const Status: FC = () => {
       {}
       {checkStatusReponse !== undefined ? (
         <StatusInfo
-          handleGoBackClick={handleGoBack}
-          goBackText={t('go-back')}
+          handleGoBackClick={
+            checkStatusReponse ? () => Router.push('/landing') : handleGoBack
```

Comment:
Move this if condition inside `handleGoBack` and set `onGoBackClick` prop with `handleGoBack` as before. I think we should rename `handleGoBackClick` to `onGoBackClick` to be consistent with other event props naming convention.